service_systemd_linux.go

```diff
@@ -189,6 +189,27 @@ func (s *systemd) Run() (err error) {
 	return s.i.Stop(s)
 }
 
+func (s *systemd) Status() (Status, string, error) {
+	exitCode, out, err := runWithOutput("systemctl", "is-active", s.Name)
+	if exitCode == 0 && err != nil {
+		return StatusError, out, err
+	}
+
+	if strings.HasPrefix(out, "active") {
```

Comment:
Slight preference to making these a switch, with the default case returning statusUnknown. Each case statement can have an HasPrefix expression.